DEPENDENCIES.rst

```diff
@@ -0,0 +1,139 @@
+###########
+Cyclus Core
+###########
+
+
+To see user and developer documentation for this code, please visit
+the `Cyclus Homepage`_.
+
+.. contents:: Table of Contents
+   :depth: 2
+
+************
+Dependencies
+************
+
+====================   ==================
+Package                Minimum Version
+====================   ==================
+`Cyclus`               1.4
+====================   ==================
+
+
+***********************
+Installing Dependencies
+***********************
+
+ The only Cycamore dependency is Cyclus. We support 3 main ways to install
```

Comment:
the main target of this documentation is the Cycamore contributing users,  as you only need the Cyclus lib and the headers to contribute to cycamore:
    - Source install: from git or tarball
    - conda (which install lib and headers)
    - deb (which install lib and headers)


the problem is this way to install is very specific as on the website we suggest to install Cyclus and Cycamore in a row (using any way you want), and on Cyclus repo we suggest to install Cycamore using the Github. So we don't have the exact right documentation...

also Github don't allow inclusion in .rst file as we do on the website, so I can't grab file or portion of file that contains the right content.
And I also like to limit the number of clics a user has to do to get the right instructions...